controllers/cluster-registration/cluster_registration_controller.go

```diff
@@ -236,9 +239,9 @@ func (r *RegisteredClusterReconciler) getSyncTargetLabels(cluster clusterapiv1.M
 }
 
 func (r *RegisteredClusterReconciler) getSyncTarget(locationContext context.Context, regCluster *singaporev1alpha1.RegisteredCluster) (*unstructured.Unstructured, error) {
-	logger := r.Log.WithName("getSyncTarget").WithValues("namespace", regCluster.Namespace, "name", regCluster.Name, "cluster", regCluster.Annotations["clusterName"])
+	logger := r.Log.WithName("getSyncTarget").WithValues("namespace", regCluster.Namespace, "name", regCluster.Name, "cluster", regCluster.ClusterName)
```

Comment:
I don't believe we want to make this change. Please see [this slack message](https://kubernetes.slack.com/archives/C021U8WSAFK/p1658737235461079) re: ClusterName being deprecated.